contrib/test_jenkins.sh

```diff
@@ -1194,14 +1194,14 @@ test_jucx() {
 
                         java -XX:ErrorFile=$WORKSPACE/hs_err_${BUILD_NUMBER}_%p.log  \
                                 -XX:OnError="cat $WORKSPACE/hs_err_${BUILD_NUMBER}_%p.log" \
-			         -cp "bindings/java/src/main/native/build-java/*" \
+			         -cp "bindings/java/resources/:bindings/java/src/main/native/build-java/*" \
```

Comment:
why need both paths in cp?

Reply:
So now ucx binaries (libucp, libuct, etc.) will not be inside a jar. They'll be inside resources folder.  In jar there would be only libjucx.so. Need to provide path to ucx binaries

Reply:
How Java classpath is affecting UCX binaries?
shouldn't we use LD_LIBRARY_PATH or rpath?

Reply:
https://github.com/openucx/ucx/blob/master/bindings/java/src/main/java/org/openucx/jucx/NativeLibs.java#L51

It's better to load UCX binaries from classpath, rather then `LD_LIBRARY_PATH`, because we can run jucx with UCX version other then globally installed, without requiring to change LD_LIBRARY_PATH everywhere

Reply:
And this is test for pull request. Maven collects binaries from all `.libs` directories that were builded: https://github.com/openucx/ucx/blob/master/bindings/java/pom.xml.in#L44-L48

We need to test with this binaries, not that ones that may be in LD_LIBRARY_PATH on this machine

Reply:
ok